crates/cli/src/cli.rs

```diff
@@ -229,6 +232,12 @@ fn main() -> anyhow::Result<()> {
         Emoji("🔍", "")
     );
 
+    // Create and configure a progress bar
+    let pb_verification = ProgressBar::new_spinner();
+    pb_verification.set_style(_spinner_style);
+    pb_verification.enable_steady_tick(Duration::from_millis(100));
```

Comment:
Is it possible to have the progress bar / spinner be based on the verification process? As in can we have it load as long as we are polling the backend for the status?

Reply:
> Is it possible to have the progress bar / spinner be based on the verification process? As in can we have it load as long as we are polling the backend for the status?

It should work as you want. Did I miss something?

Reply:
> Is it possible to have the progress bar / spinner be based on the verification process? As in can we have it load as long as we are polling the backend for the status?

It should work as you want. Did I miss something?

Reply:
Ah you're right, I misunderstood!